### ISSUE TITLE
Run containers in order

### ISSUE THREAD

**Issue by kvokka** [windmilleng/tilt#2273]
## Info

There is no any specific handlers in Tilt to manage some jobs running order, while the tools like Garden have them. For example wait until DB migration successfully ends and only after it run new deployment. With Tilt you just push the manifests to k8s and let k8s do his job.

Partially, this can be sorted out with Helm hooks feature, but it is extra dependency, right?

A few days ago I was surprised, that there is no the information about [this](https://medium.com/@kvokka/managing-db-migrations-with-kubernetes-365a9f8596c6) hack, which is absolutely natural for k8s and devaluate this feature in the other tools.

For me this feature was the point some time ago, so this hack may be useful for others. Feel free to add it to the FAQ.

Thank you for the great tool :)

**Comment by jazzdan** [comment]
Hi @kvokka, thanks for the example! I'm going to close this as a duplicate of https://github.com/windmilleng/tilt/issues/1846.
